### ISSUE TITLE
opt_flow.py: did this run?

### ISSUE THREAD

**Issue by peepo** [Kh4n/tensorflow-experiments#1]
Would you be able to shape this up?

I did find: https://github.com/commaai/speedchallenge/blob/master/data/train.mp4
linked as ../../speed_challenge/train.mp4

However there are other issues, for instance lines 46 & 47 report wrong type for a,b c,d

please let me know, as I and others might enjoy!

thanks

**Comment by Kh4n** [comment]
phew, wrote this a while ago! I do remember it running yes. maybe its a np change that is causing it? does the code not run anymore? if so what is the error

**Comment by peepo** [comment]
```
$ python3 opt_flow.py 
(480, 640, 3)
Traceback (most recent call last):
  File "/home/jonathan/Python/optical_flow/tensorflow_experiments-master/MotionTrackingLK/opt_flow.py", line 46, in <module>
    mask = cv2.line(mask, (a,b),(c,d), color[i].tolist(), 2)
cv2.error: OpenCV(4.8.0) :-1: error: (-5:Bad argument) in function 'line'
> Overload resolution failed:
>  - Can't parse 'pt1'. Sequence item with index 0 has a wrong type
>  - Can't parse 'pt1'. Sequence item with index 0 has a wrong type
```

**Comment by Kh4n** [comment]
fixed with 1d61439ef89ea8a7fdbc35ff8e02f92fcd030dda 

issue is new cv2.line function does not support floating point. fixed by changing these lines:
```
for i,(new,old) in enumerate(zip(good_new,good_old)):
    a,b = new.ravel()
    c,d = old.ravel()
    mask = cv2.line(mask, (int(a),int(b)),(int(c),int(d)), color[i].tolist(), 2)
    frame = cv2.circle(frame,(int(a),int(b)),5,color[i].tolist(),-1)
```

**Comment by peepo** [comment]
wfm rather pretty, clean and simple too!
I had tried probably int a etc, syntax is a real issue for me, as I don't stick to one language...
nice job.
